Assignment_1/q8.py

```diff
@@ -0,0 +1,3 @@
+x = int(input("enter the radius of circle "))
+print(f'the area of circle is ',3.14159* x**2,f'the circumference is ', 2*3.14159*x,f'the diameter is ',2*x)
```

Comment:
You can also use `pi` from `math` python library. Like this `from math import pi`